zulipterminal/ui_tools/views.py

```diff
@@ -1162,6 +1163,7 @@ def _fetch_user_data(controller: Any, user_id: int) -> Dict[str, Any]:
             }
             return display_data
 
+        pretty_date = user_friendly_time(data["last_active"])
```

Comment:
For users who have been inactive for longer than two weeks, the `data["last_active"]` is empty. This is causing your code to break for such users when I tried running it.